src/backend/main.ts

```diff
@@ -1362,6 +1366,17 @@ ipcMain.handle(
       status: 'done'
     })
 
+    // Playtime of this session in minutes. Uses hrtime for monotonic timer not subject to clock drift or sync errors
+    let sessionPlaytimeInMs: bigint | undefined = undefined
+    try {
+      sessionPlaytimeInMs = await syncPlaySession(appName, runner)
```

Comment:
We should still get the session playtime if the post fails so that the game closed track event still works

Reply:
If the player is not logged in, how does it fail? Does it throw or is it stuck waiting?

Reply:
I dont think it will throw anything tbh becase I dont see how it might fail. I think I can revert this to the original code.

Reply:
I was thinking something more like this https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/1165 so that we still send the game closed event even if they aren't logged in